templates/paragraphs/txex_paragraph_43/paragraph--txex-paragraph-43.html.twig

```diff
@@ -0,0 +1,19 @@
+{% extends "paragraph.html.twig" %}
```

Comment:
We don't write markup in the Drupal templates. We actually use the code written for Pattern Lab to render our Drupal components. So we'd do something like this:

```
{{ attach_library('d8_theme_main/accordion') }}
{% include "@molecules/accordion/accordion.twig" with {
	accordion_heading: content.field_txex_ref_rev_para_01,
	accordion_content: content.field_txex_ref_rev_para_02,
} %}
```

And actually, we have the `pl_attach-library.function.php` file in there, so we could move the `attach_library` line into the PL component, and not have to remember to include it in each Drupal template.

(Unrelated note, I realized the modifier variables weren't updated. They should actually be `accordion_heading_modifiers` and `accordion_content_modifiers` in the PL twig file.)